WordPress/src/main/java/org/wordpress/android/ui/posts/UpdateFeaturedImageUseCase.kt

```diff
@@ -0,0 +1,23 @@
+package org.wordpress.android.ui.posts
+
+import org.wordpress.android.fluxc.model.PostImmutableModel
+import org.wordpress.android.fluxc.model.PostModel
+import org.wordpress.android.ui.posts.EditPostRepository.UpdatePostResult
+import org.wordpress.android.ui.posts.EditPostRepository.UpdatePostResult.Updated
+
+class UpdateFeaturedImageUseCase {
```

Comment:
```suggestion
import javax.inject.Inject

class UpdateFeaturedImageUseCase @Inject constructor() {
```

I'm only able to get the PR to build if I use the `@Inject` annotation here, let me know if this looks right to you!

Reply:
Good catch. Thank you 🙇🏾

Reply:
@SiobhyB I resolved this here 610348e and did another run-through of the PR. I will be adding PR notes later with the testing instructions so this can be reviewed and merged in.